src/main/java/io/vertx/codegen/DataObjectModel.java

```diff
@@ -521,15 +521,10 @@ private void processMethod(String name, ExecutableElement getterElt, ExecutableE
       case ENUM:
         jsonifiable = true;
         break;
-      case DATA_OBJECT:
-        jsonifiable = ((DataObjectTypeInfo)propType).isSerializable();
-        break;
       case OTHER:
-        if (propType.getName().equals(Instant.class.getName())) {
-          jsonifiable = true;
-          break;
-        }
-        return;
+        jsonifiable = propType.getName().equals(Instant.class.getName()) ||
```

Comment:
Can we register `Instant` directly as a `ClassTypeInfo` with `DataObject` to avoid this hack?

Reply:
yes, that will be another PR, this PR shall also handle most java.time types and possibly Buffer